clang/lib/Driver/ToolChains/HLSL.cpp

```diff
@@ -243,6 +243,20 @@ HLSLToolChain::TranslateArgs(const DerivedArgList &Args, StringRef BoundArch,
   // FIXME: add validation for enable_16bit_types should be after HLSL 2018 and
```

Comment:
Is this FIXME comment outdated by this PR?

Reply:
Yes, I'll remove the comment.